model/portableElement/storage/PortableElementRegistry.php

```diff
@@ -534,13 +532,16 @@ public function export(PortableElementObject $object)
 
         $filesystem = $this->getFileSystem();
         foreach ($files as $file) {
-            if (strpos($file, './') === 0) {
-                //only export the files that are in the portable element package (exclude the shared libraries)
+            try {
                 $zip->addFromString($file, $filesystem->getFileContentFromModelStorage($object, $file));
+            } catch (PortableElementFileStorageException $e) {
+                // do not include missing/sharedClientLib files
```

Comment:
shouldn't we log it? e.g:

`$this->logger->info('The Portable Element ${Name} is missing. It will not be exported.');`

Reply:
I do not see much reason to log that cause it is kinda expected that some assets are not a part of a PCI code,
any real error if a required file is missing will be highlighted with detailed message when trying to import a package, and might be debugged as well,
also a registry is not a service so not easy to inject logger there